infra/compile.py

```diff
@@ -509,6 +520,10 @@ def compile_expr(tree, ctx):
             return "Error"
         elif tree.id in OUR_CONSTANTS:
             return "constants.{}".format(tree.id)
+        elif tree.id == "_":
```

Comment:
what is this feature?

Reply:
Oh lol. I sometimes use _ as an “ignore” argument (it’s a convention, not a language feature) but JS doesn’t like the same variable being bound twice in the same assignment, so I compile every _ variable to a different variable name in JS. This would be a problem if we ever wanted to use this variable, but the whole point of an “ignore” variable is that we won’t.